patt/src/main/java/org/example/FactorMethod/Logger.java

```diff
@@ -0,0 +1,4 @@
+package org.example.FactorMethod;
+public interface Logger {
+    void createLogger();
```

Comment:
N: This is Logger class. So Logger should have log(), write() or smth like that.
Responsibility of creating logger should be in LoggerFactory
(maybe you confused with method naming)